### PR TITLE
Extract history function

### LINKED ISSUE #42: Extract history to function
Tasks:
* [x] Extract file history attribute to function
* ~[ ] Move set history to end of `process()` to group `sman` I/O operations (for future fixes)~
* ~[ ] Move set CF conventions to end of `process()`~

Unit tests can likely be skipped as the code is very small.

### PR DESCRIPTION
**Pull request by truth-quark**
Closes #42.

This PR extracts & updates the file history attribute. String formatting has been updated with modern Python, and the hard-coded filename has been removed.

Known problems:
* The new version number needs to be determined (the method should ideally be decided in this PR).

### CODE REVIEW

File: umpost/um2netcdf.py [hunk 1]
```diff
@@ -603,6 +604,16 @@ def check_pressure_warnings(need_heaviside_uv, heaviside_uv, need_heaviside_t, h
               "These fields will be skipped")
 
 
+def add_global_history(infile, iris_out):
+    version = -1  # FIXME: determine version
```
Comment (blimlim) [inline umpost/um2netcdf.py:608]: Once the initial version number is decided, would it be moved to a constant to make it easier to remember to change when there are new updates?

**Review by blimlim — APPROVED** [review]
Changes look good to me!

File: umpost/um2netcdf.py (hunk 1 above)
Comment (truth-quark) [reply umpost/um2netcdf.py:608]: I suspect we'll use a versioning utility to auto-handle this (I'll add an issue now).

**Review by utkarshgupta95 — APPROVED** [review]
LGTM

**Pull request merged**